crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -210,43 +210,75 @@ fn assert_component_access_compatibility(
            query_type, filter_type, system_name, accesses);
 }
 
-/// Use to safely access and interact with up to 8 mutually exclusive [`SystemParam`]s such as
-/// two queries that reference the same mutable data  or a mutable query and acess to the ecs world.
+/// A collecton of potentially conflicting [`SystemParam`]s allowed by disjoint access.
 ///
-/// Access to each individual [`SystemParam`] by using the funcions ```p0()```, ```p1()```, ..., ```p7()```. This ensures that
-/// there's either only one mutable reference to a parameter at a time or an infinite* number of inmutable references.
-/// # Example
-/// ```
-/// use bevy_ecs::prelude::*;
+/// Allows to safely access and interact with up to 8 mutually exclusive [`SystemParam`]s such as
+/// two queries that reference the same mutable data  or a mutable query and acess to the ecs world.
 ///
-/// #[derive(Component)]
-/// struct Health(u32);
+/// Access to each individual [`SystemParam`] by using the functions `p0()`, `p1()`, ..., `p7()`. This ensures that
```

Comment:
```suggestion
/// Each individual [`SystemParam`] can be accessed by using the functions `p0()`, `p1()`, ..., `p7()`, according to the order they are defined in the `ParamSet`. This ensures that
```

I think using this wording is an improvement because it puts the `ParamSet` as the subject of the sentence (instead of the reader). Also I added a little bit of text that shows what p0 to p7 refer to.